scripts/verify_hutch

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+# function that takes in a string and verifies its a known hutch
+
+usage(){
+cat << EOF
+usage: ${BASH_SOURCE[0]} <hutch> [option]
+
+Script that returns confirmation if the passed hutch is a known/existing hutch, success indicated by exit 0, failure by exit 1. 
+
+EOF
+}
+
+foundHutch=1
+hutch=${1,,}
+if [ $# -lt 1 ]; then
+   echo 'need arguments: input hutch name' >&2
+   usage
+   exit 1
+fi 
+
+if [[ ($1 == "--help") || ($1 == "-h") ]]; then
+   usage
+   exit 0
+fi
+for i in "tmo" "txi" "rix" "xpp" "xcs" "mfx" "cxi" "mec" "ued" "ued" "det" "lfe" "kfe" "tst" "las" "hpl"; do
+    if [[ $hutch != $i ]]; then
+        foundHutch=1
+    elif [[ $hutch == $i ]]; then
+        foundHutch=0
+	exit 0
+    else 
+        foundHutch=1
+    fi
+done
+if [[ $foundHutch == 1 ]]; then
+    exit 1
+fi
```

Comment:
This section can be condensed a good amount. Try to find which lines aren't necessary and take them out :)